Detectors/TPC/reconstruction/src/TPCCATracking.cxx

```diff
@@ -95,12 +96,13 @@ int TPCCATracking::runTracking(TChain* inputClustersChain, const TClonesArray* i
     if (inputClustersChain) {
       inputClustersChain->GetEntry(iChunk);
     }
-    for (Int_t icluster = 0; icluster < inputClustersArray->GetEntries(); ++icluster) {
-      Cluster& cluster = *static_cast<Cluster*>(inputClustersArray->At(icluster));
+    for (Int_t icluster = 0; icluster < inputClustersArray->size(); ++icluster) {
```

Comment:
Can this be changed to a range based loop?

Reply:
will do

Reply:
done